ballast_info.py

```diff
@@ -67,10 +69,42 @@ def ballast_info(glider_datasets, threshold=420):
     
     for name, ds in ds_dict.items():
     
-        #Max and min ballast values per mission and total dives
+        #Max and min ballast values per mission and total dives + total volume pumped for mission
         max_ballast.append(np.nanmax(ds.ballast_pos))
         min_ballast.append(np.nanmin(ds.ballast_pos))
         total_dives.append(ds['dive_num'].values.max())
+
+        max_depth.append(int(np.nanmax(np.abs(ds.depth))))
+        
+        if np.diff(ds.time).mean()<10**8:
```

Comment:
Careful with time! This assumes a time unit (I guess nanoseconds?) It will break if time is in a different e.g. milliseconds. Probably want something like mean(diff(time)) > 1 second